script.js

```diff
@@ -1,11 +1,24 @@
 // Create an array called `colors` that contains three colors.
+var colors = ["blue","red","yellow"]
 
 // Log the last item in the array to the console.
+console.log (colors[0])
 
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+for (var i = 0; i < colors.length; i++) {
+  console.log ("My favorite color is" + " " + (colors[0]));
+  console.log ("My favorite color is" + " " + (colors[1]));
+  console.log ("My favorite color is" + " " + (colors[2]));
+}
```

Comment:
This prints out every item in the array for every iteration of the loop. You need to use `i` to print out the item that matches with the current index.